sos/report/plugins/__init__.py

```diff
@@ -1516,6 +1516,8 @@ def time_filter(path):
                     # in the corner case we just reached the sizelimit, we
                     # should collect the whole file and stop
                     limit_reached = (sizelimit and current_size == sizelimit)
+
+            _spec_tags = list(set(_spec_tags))
```

Comment:
Why this list unification? (well, it doesnt make sense to have a tag multiple times - but then we can use a set type directly..?)

Reply:
Yeah, it's just to scrub duplicates. The various tag-adding bits allow for strings or lists of strings, so doing the reduction here I feel is easier than transitioning the methods to using sets directly.